web/app/themes/justicejobs/src/js/job-search-form.js

```diff
@@ -16,15 +16,13 @@ jQuery(document).ready(function ($) {
     var str;
     var thisRadius;
 
-
-    // user view preference default
-    var userViewPref = 'list';
-
```

Comment:
Don't we want the user preference to be a list by default?

Reply:
this default wasnt actually being used. The user will see list view as default (just by css). Then in JS if local storage has a value it will then set view.